tools/partutil/check_err.go

```diff
@@ -0,0 +1,12 @@
+package partutil
+
+import "log"
+
+//Check check whether err is returned and print error message
```

Comment:
It is okay to skip the func Name. You can just start with `Checks whether err is ...`